e2e/app/page/google-login.ts

```diff
@@ -118,32 +114,34 @@ export default class GoogleLoginPage extends BasePage {
         console.error('Google login button not found. ' + err);
         throw err;
       });
-      await this.clickAndWait(googleLoginButton);
+      await BaseElement.asBaseElement(this._page, googleLoginButton).clickAndWait();
 
       if (!user || user.trim().length === 0) {
         console.warn('Login user email: value is empty!!!')
       }
       await this.enterEmail(user);
+      await this._page.waitFor(1000); // reduce chances of getting Google login recaptcha
```

Comment:
@gjuggler Have you been in touch with the recaptcha folks at all on what the best practices for e2e test suites is? This seems like Whack-a-Mole, though I don't know a better way to do it. (unless we send the captchas to a Slack channel or something).

Reply:
Is this because you can't wait for recaptcha itself to load? Anywhere I see a hardcoded wait, I get nervous.